sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -279,6 +279,7 @@ export class ReceiverImpl<ReceivedMessageT extends ReceivedMessage | ReceivedMes
    * Returns a promise that resolves to an array of messages based on given count and timeout over
    * an AMQP receiver link from a Queue/Subscription.
    *
+   * The `maxWaitTimeSeconds` provided via the options overrides the `timeoutInMs` provided in the `retryOptions`.
```

Comment:
I didn't catch this before, but I think this should be `maxWaitTimeInSeconds`. This would match what we have in Event Hubs (https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/subscribeoptions?view=azure-node-latest#maxwaittimeinseconds) and our guidelines (https://azure.github.io/azure-sdk/typescript_design.html#ts-options-suffix-durations)

Reply:
Separate PR.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/8165